pfs.ipmu.jp/usage.rst

```diff
@@ -0,0 +1,326 @@
+* PFS GitHub is open to the public, so you can read the contents and add comments except for those in the private repositories, but you cannot push to git repositories unless you follow the above procedure.
+* PFS JIRA is open to the public, so you can read the contents, but you cannot make any commitments therein unless you follow the above procedure.
+* In the PFS GitHub organization, there is one team per institute where the members are given admin privilege. So instead of sending email to github at pfs.ipmu.jp, you can also ask the members in your institute to add yourself with privilege given to repositories related to your institute. 
+
+======
+When you lost your account information: 
+======
+
+SuMIRe/PFS PBworks wiki
+  Access `password reset <https://my.pbworks.com/?p=forgot>`_ and follow an instruction shown.
+pfs.ipmu.jp (https)
+  Send email to pfs at pfs.ipmu.jp from email address you registered.
+PFS JIRA
+  If you have an account under single sign on at pfs.ipmu.jp (https), 
+  send email to pfs at pfs.ipmu.jp. Otherwise access 
+  `JIRA registration page <https://pfspipe.ipmu.jp/jira/secure/ForgotLoginDetails.jspa>`_
+  and follow an instruction shown. 
+
+
+Details of the services
+******
+
+SuMIRe/PFS PBworks wiki
+------
+
+"SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
+
+======
+Login and ToC
+======
+
+After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
+you will get a project index page named 
+"Subaru Measurement of Images and Redshifts (SuMIRe)". 
+This page contains links to contents in the wiki, such as 
+
+* PFS Project Office (General links, including link to telecon indexes)
+* Documents (Link list to documents)
+* Meeting, conference, etc.  (Link list to meetings)
+* Mailing lists (List of avail lists)
+* PFS working groups (Member list)
+
+======
+Page indexes
+======
+
+For some continuous meetings, index for each agenda/memo pages are avail.
+
+* `Systems Engineering group telecon <http://sumire.pbworks.com/System-engineer-group-telecon>`_
+* `Manager group telecon <http://sumire.pbworks.com/Manager-group-telecon>`_
+
+
+======
+Files
+======
+
+Also, you can find all files uploaded into this SuMIRe/PFS PBworks from 
+`folder listing <http://sumire.pbworks.com/w/browse/#view=ViewAllFiles>`_.
+Some files are categorized into *FOLDERS*, and you can get each list by 
+clicking FOLDER name at left side. 
+
+======
+Editing manual
+======
+
+You can find `manual for editing PBworks <http://usermanual.pbworks.com/>`_. 
+
+
+Public Web Site at pfs.ipmu.jp
+------
+
+Public project information is at http://pfs.ipmu.jp , 
+such as list of meetings, list of publications, and instrument parameters. 
+
+Internal Web Site -- pfs.ipmu.jp (https)
+------
+
+Every contents at https://pfs.ipmu.jp/ are project only, and you will 
+be required to log in to view pages. 
+
+If you have any issue on this site, contact administrator 
+at pfs@pfs.ipmu.jp . 
```

Comment:
fixed two, but is it better to make them as link like line 257? (anyway, let have another ticket if we want).